valve-monitor/web-app/src/constants/ui.ts

```diff
@@ -12,12 +10,15 @@ const ERROR_MESSAGE = {
     "An internal error occurred. If this problem persists, contact <a href='https://discuss.blues.io' target='_blank' rel='noreferrer'>Blues Support</a>.",
   UNAUTHORIZED:
     "Authentication failed. Please ensure you have a valid HUB_AUTH_TOKEN environment variable.",
-  DEVICE_NAME_CHANGE_FAILED:
-    "An error occurred changing the name.",
+  DEVICE_NAME_CHANGE_FAILED: "An error occurred changing the name.",
   DATABASE_NOT_RUNNING:
     "Can't reach the database server. Please make sure your database is properly connected.",
   NO_PROJECT_ID:
     "A project ID is required, but none was found. Ensure your environment variables are configured correctly for HUB_PROJECTUID.",
+  UPDATE_FLEET_MONITOR_FREQUENCY_FAILED:
+    "Failed to update the fleet’s flow monitoring frequency. If this problem persists, contact Blues support.",
+  UPDATE_ALARM_THRESHOLD_FAILED:
```

Comment:
Would it make sense to have this error named `UPDATE_FLEET_ALARM_THRESHOLD_FAILED`, since this env var can be set at the device level too?

Reply:
Ah yeah, and I did it in the one right above. Will fix.